settings.py

```diff
@@ -90,8 +93,8 @@
     if not hasattr(settingsmodule, 'PROTOCOL'):
         setattr(settingsmodule, 'PROTOCOL', 'http')
 
-if WEB_IMAGE_UPLOAD_PATH and not os.path.exists(WEB_IMAGE_UPLOAD_PATH):
-    os.mkdir(WEB_IMAGE_UPLOAD_PATH)
+# if WEB_IMAGE_UPLOAD_PATH and not os.path.exists(WEB_IMAGE_UPLOAD_PATH):
```

Comment:
Why are we commenting this out?

Reply:
ah damn yes! was erroring out when run locally